examples/xml-operations/xml_operations.md

```diff
@@ -1,13 +1,10 @@
 # XML operations
 
-- `+` does concatenation. 
-- `==` does deep equals. 
-- `foreach` iterates over each item.
-- `x[i]` gives i-th item (empty sequence if none).
-- `x.id` accesses required attribute named `id`: result is error if there is no such attribute or if x is not a singleton.
-- `x?.id` accesses optional attribute named `id`: result is `()` if there is no such attribute. Langlib `lang.xml` provides other operations.
-- Mutate an element using `e.setChildren(x)`.
+In XML, It is possible to perform concatenation and deep equals checks.
```

Comment:
we might need to say what deep equality means with XML? or we should have a related link pointing to something(BBE?) that explains deep equality? WDYT @hasithaa

Reply:
I think best option is to create a new BBE covering this. @DamithaSenevirathne Please create an issue.